crds/virtualimage.yaml

```diff
@@ -188,7 +191,9 @@ spec:
               oneOf:
                 - properties:
                     storage:
-                      enum: ["ContainerRegistry"]
+                      enum:
+                        - "ContainerRegistry"
+                        - "Kubernetes"
```

Comment:
Something is wrong here. This `oneOf` is useless here, you can set `required` for the `storage` field and thats it. Also, see history for this CRD, there should be and old version of this `oneOf`.

Reply:
Sorry. I don't understand what I have to do.

Reply:
Fixed.